src/sentry/static/sentry/app/components/letar.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+
+const Letar = React.createClass({
+  propTypes: {
+    user: React.PropTypes.object.isRequired
+  },
+
+  COLORS: {
+    blue: '#25A6F7',
+    green: '#6FBA57',
+    orange: '#EC5B33',
+    red: '#E35141',
+    pink: '#E56AA6',
+    purple: '#836CC2'
+  },
```

Comment:
yeah, i should check in with @ckj generally to make sure everything looks ok so will ask about this too